src/artemis/tests/test_log.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+from artemis.log import LOGGER, set_up_logging
+
+
```

Comment:
I have been working on adding the first two requests, unfortunately the singleton nature of loggers results in some things persisting beyond each individual test if you're not careful. So this has taken a bit longer than expected.

For the last point, the system doesn't appear to be affected by the connection to graylog at all. If graylog isn't running when it tries to send a log it just doesn't get sent and carries on as normal. The log message is present in the stdout and logfile so it is not lost entirely. Obviously i've not been able to test this completely as it only goes a certain way through when you set it off on a scan on a local work station but "starting scan with..." log message appears on the terminal and it doesn't fall over if there is no local graylog running to send it to (when run on dev). This suggests it should behave the same for prod, since i'ts just a different host and port.